### ISSUE TITLE
Feature request - Offline solution

### ISSUE THREAD

**Issue by basinger** [dbaldwin/DroneBlocks-Web#17]
Currently I need to be online to load the mission. What if I am somewhere where there is no connection? It would be great if the missions could be loaded onto the device and then without internet connection be executed.

**Comment by sp00x** [comment]
Definitely +1 on this one. I just started exploring the app, and that was one of the things I noticed immediately was kind of a "huh, really?" aspect. I get it for sharing missions etc, but that's just a part of what this is about.

**Comment by gateway** [comment]
Any updates to this.. Ill be trying to take some flatland photos at this years Burning Man.. getting signal out there is slim to none.

**Comment by iHaagcom** [comment]
I agree with this completely. An offline solution will help in the Classroom Environment as well as no everyone has 4G + Wifi.

**Comment by pilotmediaLLC** [comment]
Here it is 2020 and it is still impossible to fly without an internet connection. What the heck are you supposed to do with your students in the field?  Dronebocks = poor usability quality and there's no real support for known issues. For example there is no support documentation for simple issues like "Why doesn't a mission button appear in my menu?"  "How can I fly without an internet connection?" 
3 years after this thread was posted - same-same -- no documentation, no response.
